src/api/accessToken.ts

```diff
@@ -33,4 +33,15 @@ accessTokensRouter.get('/', (req, res) => {
   res.json(tokens)
 })
 
+accessTokensRouter.delete('/:id', (req, res) => {
+  const tokens = getAccessTokens()
+  tokens.forEach((e, i) => {
```

Comment:
`tokens.find((e,i)=> {.....` -> This way you can have less than O(n) instead of forsure O(n)